scripts.js

```diff
@@ -178,3 +180,54 @@ closePopupBtn.forEach((btn, index) => {
     popups[index].classList.remove("open-popup");
   });
 });
+// form validation
+const form = document.querySelector("form");
+const email = document.querySelector('input[type="email"]');
+const msg = document.querySelector("form .message");
+
+function showError(m) {
+  msg.style.display = "block";
+  msg.innerText = m;
+}
+
+function showSuccess(mass) {
+  msg.style.display = "block";
+  msg.style.color = "white";
+  msg.style.backgroundColor = "green";
+  msg.innerText = mass;
+}
+
+function checkLowerCase(input) {
+  if (input.value !== input.value.toLowerCase()) {
+    showError("please write the email in lowercase");
+  } else {
+    showSuccess("Thank You.");
+    form.submit();
+  }
+}
+
+form.addEventListener("submit", (event) => {
+  event.preventDefault();
+  checkLowerCase(email);
+});
+// local storage
+const valueInput = JSON.parse(localStorage.getItem("valueInput")) || [];
+
```

Comment:
- [ ] Great work saving data to the `localStorage`. Remember that the requirements state that you should save all form data in an _**object**_, not an _**array**_ and save the object in the `localStorage`. Kindly implement this. 👍 

#### Hint: Delete this line of code